### PR TITLE
added: button component and fix: code preview

### PR DESCRIPTION
**Pull request by md-amir-gauhar**

### CODE REVIEW

**Comment by md-amir-gauhar** [comment]
Screenshot (143)

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
+.button, 
+.button.large,
+.button.button-solid {
+  border-radius: 5px;
+  border: none;
+  background-color: rgb(6, 140, 109);
```
Comment (xsaurabhsuthar) [inline components/button/button.css:6]: You can use variable for colors.

File: components/button/button.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+.button.small {
+  padding: 0.625rem;
+  font-weight: 500;
+  font-size: 0.8rem;
+}
+
+.button.mid {
+  padding: 0.8rem;
```
Comment (xsaurabhsuthar) [inline components/button/button.css:24]: Can use pixels for small size instead of rems.

**Review by xsaurabhsuthar — APPROVED** [review]

**Review by samflab — APPROVED** [review]
Make the minor changes pointed out to you earlier. Other than that, its good to go 👍🏼

File: components/button/button.css (hunk 2 above)
Comment (md-amir-gauhar) [reply components/button/button.css:24]: will do the required refactoring. Thanks:)

**Pull request merged**